GP/toy_experiments/toy_sin_noise.py

```diff
@@ -60,17 +59,17 @@
         X_init = np.random.uniform(0, 10, init_num_samples).reshape(-1, 1)  # sample 7 points at random from the bounds to initialise with
         plot_sample = np.linspace(0, 10, 50).reshape(-1, 1)  # samples for plotting purposes
 
-        Y_init = linear_sin_noise(X_init, noise_coeff, plot_sample, coefficient, modification, fplot=False)
+        Y_init = linear_sin_noise(X_init, noise_coeff, plot_sample, coefficient, modification, fplot=True)
```

Comment:
I think using fplot is good for when we want to adjust the objective. We can set it to false once we decide on an appropriate form!